src/dispatchers/keyDispatcher.ts

```diff
@@ -0,0 +1,79 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export type KeyCallback = (keyCode: number) => any;
+
+  export class Key {
+    private static _DISPATCHER_KEY = "__Plottable_Dispatcher_Key";
+    private _connected = false;
```

Comment:
Can we attach the comment here that the parent class var name is going to be changed or similar?